### ISSUE TITLE
Remix: Azure Functions + SWA

### ISSUE THREAD

**Issue by gvillo** [aaronpowell/azure-remix-stack#15]
Hi @aaronpowell! wondering if there is an example in Microsoft using Remix w/Azure Functions and Static Web Apps, I found this adapter (https://github.com/scandinavianairlines/remix-azure-functions) useful, but I have no idea how it's this working, if it has cold start and the costs involved in a regular Remix SSR app. 

It would be awesome if this azure-remix-stack got updated with all of that, because I am trying to move from NextJS Container Apps to Remix Azure SWA+Functions, but I am not 100% sure that this move will work ok.

Thanks!

**Comment by aaronpowell** [comment]
No, I don't have any info on this, and I'm not actively working in this space anymore so I probably should archive this repo.
